src/drop-down-menu.jsx

```diff
@@ -211,17 +212,19 @@ const DropDownMenu = React.createClass({
         }
       }
     }
-
     let selectedItem = this.props.menuItems[selectedIndex];
     if (selectedItem) {
-      displayValue = selectedItem[labelMember];
+      displayValue = selectedItem[labelMember] || selectedItem[displayMember];
```

Comment:
Yes, exactly.
So, sorry, but, I don't understand :grimacing: in which case is this useful?